pkgs/development/libraries/aribb25/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchFromGitLab
+, fetchpatch
+, autoreconfHook
+, pkgconfig
+, pcsclite
+, PCSC
+}:
+
+stdenv.mkDerivation rec {
+  pname = "aribb25";
+  version = "0.2.7";
+
+  src = fetchFromGitLab {
+    domain = "code.videolan.org";
+    owner = "videolan";
+    repo = pname;
+    # rev = version; FIXME: uncomment in next release
+    rev = "c14938692b313b5ba953543fd94fd1cad0eeef18"; # 0.2.7 with build fixes
+    sha256 = "1kb9crfqib0npiyjk4zb63zqlzbhqm35nz8nafsvdjd71qbd2amp";
+  };
+
+  nativeBuildInputs = [ autoreconfHook pkgconfig ];
+  buildInputs = if stdenv.isDarwin then [ PCSC ] else [ pcsclite ];
+
+  patches = let
+    url = pr: "https://code.videolan.org/videolan/${pname}/-/merge_requests/${pr}.patch";
```

Comment:
This type of patch is unstable, it might change when the MR changes. That would then break the fixed-output-derivation. You should refer to the relevant commits directly instead.

Reply:
Fixed.